src/__stories__/header-story.tsx

```diff
@@ -55,6 +55,35 @@ export const Default: StoryComponent = () => {
 
 Default.storyName = 'Header';
 
+export const NoHeader: StoryComponent = () => {
+    const [extraSideBySide, extraSideBySideCheckbox] = useCheckbox(
+        'Extra content placed on the right in desktop',
+        true
+    );
+    return (
+        <Stack space={16}>
+            <div data-testid="header-layout">
+                <HeaderLayout
+                    isInverse
+                    sideBySideExtraOnDesktop={extraSideBySide}
+                    breadcrumbs={
+                        <NavigationBreadcrumbs
+                            title="Bills"
+                            breadcrumbs={[{title: 'Account', url: '/consumptions'}]}
+                        />
+                    }
+                    extra={<Placeholder />}
+                />
+            </div>
+            <ResponsiveLayout>
+                <Stack space={16}>{extraSideBySideCheckbox}</Stack>
+            </ResponsiveLayout>
```

Comment:
I'd use story args instead, see other stories, like card stories for example

Reply:
Yes, but as the story is placed in the same file, I prefer to migrate it completely, not only this new story and it can be done in a maintenance PR if possible ;)

Reply:
fair